package proto_ff;

import "yd_fieldoptions.proto";


message buttonbutton
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "按钮id"];
	optional int32 time = 2[(yd_fieldoptions.field_cname) = "读条时间"];
}


message Sheet_buttonbutton
{
	repeated buttonbutton buttonbutton_List = 1[(yd_fieldoptions.field_arysize)=80];
}
